### ISSUE TITLE
526: Show error links on review & submit

### ISSUE THREAD

**Issue by Mottie** [department-of-veterans-affairs/va.gov-team#36700]
## Description

Form 526 shows a generic error message when validation fails upon submission. We need to enable the error links which provide a path for the Veteran to the appropriate accordion & entry that needs editing.

## Tasks

- [ ] Enable error links
- [ ] Unit tests

## Definition of done

- [ ] Error links showing in production
- [ ] All tests passing
